native/cocos/platform/SDLHelper.cpp

```diff
@@ -223,9 +223,6 @@ void SDLHelper::dispatchSDLEvent(uint32_t windowId, const SDL_Event &sdlEvent, b
 
     switch (sdlEvent.type) {
         case SDL_QUIT: {
```

Comment:
Whether SDL_QUIT is the last message to be processed in the message loop

Reply:
SDL_QUIT is the last event customized by SDL, which can be handled or not according to the requirements